apps/meteor/ee/app/livechat-enterprise/server/hooks/checkAgentBeforeTakeInquiry.ts

```diff
@@ -55,25 +52,23 @@ const validateMaxChats = async ({
 	});
 
 	if (maxNumberSimultaneousChat === 0) {
-		cbLogger.debug(`Callback success. Agent ${agentId} max number simultaneous chats on range`);
+		cbLogger.info(`Chat can be taken by Agent ${agentId}: max number simultaneous chats on range`);
```

Comment:
```suggestion
		cbLogger.debug(`Chat can be taken by Agent ${agentId}: max number simultaneous chats on range`);
```

I feel we should keep this one on a lower level since this will get executed for each new room, leading to more logs for larger operations

Reply:
I'll concede u this one, this was not intended to be info, just the other one lol